### ISSUE TITLE
Pros/Cons vs meteor-webpack

### ISSUE THREAD

**Issue by Morganjackson** [jcoreio/crater#78]
> It's 2016, and your Meteor app has crash landed in  ...

This describes my situation perfectly!

What do you see the pros/cons of using crater vs using [meteor-webpack](https://github.com/thereactivestack/meteor-webpack) & [meteor-react-router-ssr](https://github.com/thereactivestack/meteor-react-router-ssr)?

**Comment by darkadept** [comment]
I tried using meteor-webpack before and it was really slow, especially during development. I also ran into the problem where it forced you to use certain package versions and you had to hack the internals to change it. 

What I love about Crater is that my app is made up of my code and standard npm packages with Meteor thrown in as flavor. I can use as little or as much of Meteor that I want. I just import the parts that I want to use.

It wouldn't be too hard to transition my app code away from Meteor in the future, if needed. I'm using it as my backend, user authentication, reactive data source and RPC. I could transition to using a REST API just by changing my Redux actions and reducers.

**Comment by jedwards1211** [comment]
> I tried using meteor-webpack before and it was really slow, especially during development. I also ran into the problem where it forced you to use certain package versions and you had to hack the internals to change it.

Yeah, dealing with Meteor package versions has been a nightmare in my experience.  NPM made some hugely helpful decisions to try to prevent dependency hell.

I haven't tried those meteor packages so I can't speak to them too specifically...but I can say for sure, this package is fully customizable (though that comes with a downside -- dealing with conflicts when pulling updates from this package can be a pain).

**Comment by jedwards1211** [comment]
I've been thinking about making a meteor2npm package converter.  But then again, I've been thinking about trying some standalone DDP and Mongo oplog tailing tools without Meteor too

**Comment by jedwards1211** [comment]
@Morganjackson the biggest con for crater right now is that some Meteor packages don't work with it as-is (e.g. react-meteor-data).

**Comment by Morganjackson** [comment]
@jedwards1211 ok thanks for the comments! 

I have been trying to use NPM packages in place of atmosphere anyway so I havent had any issues with Meteor packages yet.
   
Have you posted this in crater.io or the forums yet? I was surprised I hadn't seen it around when I found it on github.

**Comment by jedwards1211** [comment]
No, I've mainly posted about it on issues about isobuild slowness or related problems.  I'm tempted to wait until I have some generic solution for supporting meteor packages before posting to the forums.

**Comment by jedwards1211** [comment]
@Morganjackson plus, at some point I want to experiment with writing a tool that builds meteor packages for requiring via npm without using meteor's dev tools at all.  If that's fully possible, it would be an even more lightweight approach than this project.

**Comment by john-osullivan** [comment]
No need to close this conversation as other people might find it useful, but this repository is now the only quality Meteor + React + Webpack boilerplate: meteor-webpack & meteor-react-router-ssr are no longer being actively maintained.  

I'm in for a good amount of work this weekend switching to your implementation, @jedwards1211 !  Thanks for building Crater, it looks like a great piece of work.

**Comment by jedwards1211** [comment]
@john-osullivan Thanks!  Though I'm afraid I'm also slowly moving on from this, as my company is moving away from using Meteor in contract projects.  But I'll try to keep up with maintenance.

**Comment by john-osullivan** [comment]
Bummer, but good to know!  Poor Meteor, webpack is really eating isobuild's lunch.

**Comment by jedwards1211** [comment]
Yep, and npm is eating its package system
